roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,20 +15,27 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
+            "has_registry_console": False,
```

Comment:
My thought is that it may be clearer, in this particular case, to have a literal list of the images instead of doing some form of computation to get to them.